src/libAtomVM/term.h

```diff
@@ -519,11 +527,12 @@ static inline int term_is_reference(term t)
 }
 
 /**
- * @brief Checks if a term is a function
+ * @brief Checks if a term is a fun
  *
  * @details Returns 1 if a term is a fun, otherwise 0.
  * @param t the term that will be checked.
  * @return 1 if check succeeds, 0 otherwise.
+ * @deprecated renamed to term_is_fun.
```

Comment:
It might be nice to add a warning here at compile time. The depreciation warning in the doc may be overlooked.

Reply:
This might be a good idea for main, I would keep things as they are in release-0.6 branch.